tests/unit/commands/local/lib/test_sam_function_provider.py

```diff
@@ -516,28 +517,30 @@ def test_must_ignore_opt_in_AmazonLinux1803_layer(self):
             self.assertEqual(actual_layer, expected_layer)
 
     def test_layers_created_from_template_resources(self):
-        resources = {
-            "Layer": {"Type": "AWS::Lambda::LayerVersion", "Properties": {"Content": {"Bucket": "bucket"}}},
-            "ServerlessLayer": {"Type": "AWS::Serverless::LayerVersion", "Properties": {"ContentUri": "/somepath"}},
-        }
+        with tempfile.NamedTemporaryFile() as handle:
```

Comment:
Any reason we are not mocking out the call to check if the file exists instead of creating one? Typically unit tests do not create resources on the system.

Reply:
thanks, changed to use `@patch`